shapes/plot-description/slope-type/valid.ttl

```diff
@@ -0,0 +1,30 @@
+
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX sosa: <http://www.w3.org/ns/sosa/>
+PREFIX tern: <https://w3id.org/tern/ontologies/tern/>
+PREFIX void: <http://rdfs.org/ns/void#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+
+<https://example.com/observation/slope-type/b91c54f2-9354-43ee-8412-09974fd2c23c>
+    a tern:Observation ;
+    void:inDataset <https://example.com/dataset/1> ;
+    sosa:hasFeatureOfInterest [
+            a tern:FeatureOfInterest ;
+            void:inDataset <https://example.com/dataset/1> ;
+            tern:featureType <http://linked.data.gov.au/def/tern-cv/2cf3ed29-440e-4a50-9bbc-5aab30df9fcd>
+        ] ;
+    sosa:hasResult [
+            a
+                tern:Concept ,
+                tern:Value ;
+            rdf:value <https://linked.data.gov.au/def/test/dawe-cv/eb2cd300-a034-5d20-9504-22c19687ca57> ;
+            sosa:isResultOf <https://example.com/observation/slope-type/b91c54f2-9354-43ee-8412-09974fd2c23c> ;
+            tern:vocabulary <https://linked.data.gov.au/def/test/dawe-cv/d893e669-c530-4bc3-a057-a5799ffcb5db>
```

Comment:
Add a test to ensure the correct vocabulary value is set. Since the current URI of the vocabulary is not stable yet, let's use https://www.w3.org/TR/shacl/#PatternConstraintComponent (regex) to ensure the value at least ends with `d893e669-c530-4bc3-a057-a5799ffcb5db`.